UPDATING.md

```diff
@@ -24,6 +24,7 @@ assists people when migrating to a new version.
 
 ## Next
 
+- [-](https://github.com/apache/superset/pull/-): Removed deprecated API `/superset/tables/<int:db_id>/<schema>/...`
```

Comment:
Could you move this to the breaking changes section and add the PR number?

Reply:
this is strange I think we used to only have `Next` and potential downtime, since all that lands on `Next` would somewhat be a minor breaking change. What's the difference between `Next` and `Breaking changes` on this context?

Reply:
The difference is that in Next we add things that are not breaking changes but might be helpful when upgrading.

Reply:
I think is just a form of separating things you should really be careful when upgrading from things you might find useful or interesting.